karma.conf.js

```diff
@@ -89,7 +89,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
```

Comment:
Is this just for your testing purposes? In general, think PhantomJS would be better than Chrome since it is faster.